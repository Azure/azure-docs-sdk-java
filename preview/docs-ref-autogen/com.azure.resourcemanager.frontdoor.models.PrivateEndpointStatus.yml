### YamlMime:JavaType
uid: "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus"
fullName: "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus"
name: "PrivateEndpointStatus"
nameWithType: "PrivateEndpointStatus"
summary: "Defines values for PrivateEndpointStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateEndpointStatus extends ExpandableStringEnum<PrivateEndpointStatus>"
constructors:
- "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.PrivateEndpointStatus()"
fields:
- "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.APPROVED"
- "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.DISCONNECTED"
- "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.PENDING"
- "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.REJECTED"
- "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.TIMEOUT"
methods:
- "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.frontdoor.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-frontdoor:1.0.0-beta.1
