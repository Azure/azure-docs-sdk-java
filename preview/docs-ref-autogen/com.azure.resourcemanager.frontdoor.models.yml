### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.frontdoor.models"
fullName: "com.azure.resourcemanager.frontdoor.models"
name: "com.azure.resourcemanager.frontdoor.models"
summary: "Package containing the data models for FrontDoorManagementClient. FrontDoor Client."
classes:
- "com.azure.resourcemanager.frontdoor.models.ActionType"
- "com.azure.resourcemanager.frontdoor.models.AggregationInterval"
- "com.azure.resourcemanager.frontdoor.models.Availability"
- "com.azure.resourcemanager.frontdoor.models.Backend"
- "com.azure.resourcemanager.frontdoor.models.BackendEnabledState"
- "com.azure.resourcemanager.frontdoor.models.BackendPool"
- "com.azure.resourcemanager.frontdoor.models.BackendPoolProperties"
- "com.azure.resourcemanager.frontdoor.models.BackendPoolUpdateParameters"
- "com.azure.resourcemanager.frontdoor.models.BackendPoolsSettings"
- "com.azure.resourcemanager.frontdoor.models.CacheConfiguration"
- "com.azure.resourcemanager.frontdoor.models.CheckNameAvailabilityInput"
- "com.azure.resourcemanager.frontdoor.models.CustomHttpsConfiguration"
- "com.azure.resourcemanager.frontdoor.models.CustomHttpsProvisioningState"
- "com.azure.resourcemanager.frontdoor.models.CustomHttpsProvisioningSubstate"
- "com.azure.resourcemanager.frontdoor.models.CustomRule"
- "com.azure.resourcemanager.frontdoor.models.CustomRuleEnabledState"
- "com.azure.resourcemanager.frontdoor.models.CustomRuleList"
- "com.azure.resourcemanager.frontdoor.models.DynamicCompressionEnabled"
- "com.azure.resourcemanager.frontdoor.models.Endpoint"
- "com.azure.resourcemanager.frontdoor.models.EndpointType"
- "com.azure.resourcemanager.frontdoor.models.EnforceCertificateNameCheckEnabledState"
- "com.azure.resourcemanager.frontdoor.models.Error"
- "com.azure.resourcemanager.frontdoor.models.ErrorDetails"
- "com.azure.resourcemanager.frontdoor.models.ExperimentList"
- "com.azure.resourcemanager.frontdoor.models.ExperimentUpdateModel"
- "com.azure.resourcemanager.frontdoor.models.ForwardingConfiguration"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorCertificateSource"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorCertificateType"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorEnabledState"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorForwardingProtocol"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorHealthProbeMethod"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorListResult"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorProperties"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorProtocol"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorQuery"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorRedirectProtocol"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorRedirectType"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorResourceState"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorTlsProtocolType"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorUpdateParameters"
- "com.azure.resourcemanager.frontdoor.models.FrontendEndpointLink"
- "com.azure.resourcemanager.frontdoor.models.FrontendEndpointProperties"
- "com.azure.resourcemanager.frontdoor.models.FrontendEndpointUpdateParameters"
- "com.azure.resourcemanager.frontdoor.models.FrontendEndpointUpdateParametersWebApplicationFirewallPolicyLink"
- "com.azure.resourcemanager.frontdoor.models.FrontendEndpointsListResult"
- "com.azure.resourcemanager.frontdoor.models.HeaderAction"
- "com.azure.resourcemanager.frontdoor.models.HeaderActionType"
- "com.azure.resourcemanager.frontdoor.models.HealthProbeEnabled"
- "com.azure.resourcemanager.frontdoor.models.HealthProbeSettingsModel"
- "com.azure.resourcemanager.frontdoor.models.HealthProbeSettingsProperties"
- "com.azure.resourcemanager.frontdoor.models.HealthProbeSettingsUpdateParameters"
- "com.azure.resourcemanager.frontdoor.models.KeyVaultCertificateSourceParametersVault"
- "com.azure.resourcemanager.frontdoor.models.LatencyMetric"
- "com.azure.resourcemanager.frontdoor.models.LatencyScorecardAggregationInterval"
- "com.azure.resourcemanager.frontdoor.models.LoadBalancingSettingsModel"
- "com.azure.resourcemanager.frontdoor.models.LoadBalancingSettingsProperties"
- "com.azure.resourcemanager.frontdoor.models.LoadBalancingSettingsUpdateParameters"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleDefinition"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleEnabledState"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleExclusion"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleExclusionMatchVariable"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleExclusionSelectorMatchOperator"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleGroupDefinition"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleGroupOverride"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleOverride"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleSet"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleSetActionType"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleSetDefinitionList"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleSetList"
- "com.azure.resourcemanager.frontdoor.models.MatchCondition"
- "com.azure.resourcemanager.frontdoor.models.MatchProcessingBehavior"
- "com.azure.resourcemanager.frontdoor.models.MatchVariable"
- "com.azure.resourcemanager.frontdoor.models.MinimumTlsVersion"
- "com.azure.resourcemanager.frontdoor.models.NetworkExperimentResourceState"
- "com.azure.resourcemanager.frontdoor.models.NetworkOperationStatus"
- "com.azure.resourcemanager.frontdoor.models.Operator"
- "com.azure.resourcemanager.frontdoor.models.PolicyEnabledState"
- "com.azure.resourcemanager.frontdoor.models.PolicyMode"
- "com.azure.resourcemanager.frontdoor.models.PolicyRequestBodyCheck"
- "com.azure.resourcemanager.frontdoor.models.PolicyResourceState"
- "com.azure.resourcemanager.frontdoor.models.PolicySettings"
- "com.azure.resourcemanager.frontdoor.models.PreconfiguredEndpointList"
- "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus"
- "com.azure.resourcemanager.frontdoor.models.ProfileList"
- "com.azure.resourcemanager.frontdoor.models.ProfileUpdateModel"
- "com.azure.resourcemanager.frontdoor.models.PurgeParameters"
- "com.azure.resourcemanager.frontdoor.models.RedirectConfiguration"
- "com.azure.resourcemanager.frontdoor.models.RouteConfiguration"
- "com.azure.resourcemanager.frontdoor.models.RoutingRule"
- "com.azure.resourcemanager.frontdoor.models.RoutingRuleEnabledState"
- "com.azure.resourcemanager.frontdoor.models.RoutingRuleLink"
- "com.azure.resourcemanager.frontdoor.models.RoutingRuleProperties"
- "com.azure.resourcemanager.frontdoor.models.RoutingRuleUpdateParameters"
- "com.azure.resourcemanager.frontdoor.models.RoutingRuleUpdateParametersWebApplicationFirewallPolicyLink"
- "com.azure.resourcemanager.frontdoor.models.RuleType"
- "com.azure.resourcemanager.frontdoor.models.RulesEngineAction"
- "com.azure.resourcemanager.frontdoor.models.RulesEngineListResult"
- "com.azure.resourcemanager.frontdoor.models.RulesEngineMatchCondition"
- "com.azure.resourcemanager.frontdoor.models.RulesEngineMatchVariable"
- "com.azure.resourcemanager.frontdoor.models.RulesEngineOperator"
- "com.azure.resourcemanager.frontdoor.models.RulesEngineProperties"
- "com.azure.resourcemanager.frontdoor.models.RulesEngineRule"
- "com.azure.resourcemanager.frontdoor.models.RulesEngineUpdateParameters"
- "com.azure.resourcemanager.frontdoor.models.SecurityPolicyLink"
- "com.azure.resourcemanager.frontdoor.models.SessionAffinityEnabledState"
- "com.azure.resourcemanager.frontdoor.models.Sku"
- "com.azure.resourcemanager.frontdoor.models.SkuName"
- "com.azure.resourcemanager.frontdoor.models.State"
- "com.azure.resourcemanager.frontdoor.models.TimeseriesAggregationInterval"
- "com.azure.resourcemanager.frontdoor.models.TimeseriesDataPoint"
- "com.azure.resourcemanager.frontdoor.models.TimeseriesType"
- "com.azure.resourcemanager.frontdoor.models.Transform"
- "com.azure.resourcemanager.frontdoor.models.TransformType"
- "com.azure.resourcemanager.frontdoor.models.ValidateCustomDomainInput"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicyList"
enums:
- "com.azure.resourcemanager.frontdoor.models.ResourceType"
interfaces:
- "com.azure.resourcemanager.frontdoor.models.CheckNameAvailabilityOutput"
- "com.azure.resourcemanager.frontdoor.models.Endpoints"
- "com.azure.resourcemanager.frontdoor.models.Experiment"
- "com.azure.resourcemanager.frontdoor.models.Experiment.Definition"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages.Blank"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages.WithEnabledState"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages.WithEndpointA"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages.WithEndpointB"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.frontdoor.models.Experiment.DefinitionStages.WithTags"
- "com.azure.resourcemanager.frontdoor.models.Experiment.Update"
- "com.azure.resourcemanager.frontdoor.models.Experiment.UpdateStages"
- "com.azure.resourcemanager.frontdoor.models.Experiment.UpdateStages.WithDescription"
- "com.azure.resourcemanager.frontdoor.models.Experiment.UpdateStages.WithEnabledState"
- "com.azure.resourcemanager.frontdoor.models.Experiment.UpdateStages.WithTags"
- "com.azure.resourcemanager.frontdoor.models.Experiments"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.Definition"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.Blank"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithBackendPools"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithBackendPoolsSettings"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithEnabledState"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithFriendlyName"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithFrontendEndpoints"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithHealthProbeSettings"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithLoadBalancingSettings"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithRoutingRules"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.DefinitionStages.WithTags"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.Update"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages.WithBackendPools"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages.WithBackendPoolsSettings"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages.WithEnabledState"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages.WithFriendlyName"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages.WithFrontendEndpoints"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages.WithHealthProbeSettings"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages.WithLoadBalancingSettings"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages.WithRoutingRules"
- "com.azure.resourcemanager.frontdoor.models.FrontDoor.UpdateStages.WithTags"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorNameAvailabilities"
- "com.azure.resourcemanager.frontdoor.models.FrontDoorNameAvailabilityWithSubscriptions"
- "com.azure.resourcemanager.frontdoor.models.FrontDoors"
- "com.azure.resourcemanager.frontdoor.models.FrontendEndpoint"
- "com.azure.resourcemanager.frontdoor.models.FrontendEndpoints"
- "com.azure.resourcemanager.frontdoor.models.LatencyScorecard"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleSetDefinition"
- "com.azure.resourcemanager.frontdoor.models.ManagedRuleSets"
- "com.azure.resourcemanager.frontdoor.models.NetworkExperimentProfiles"
- "com.azure.resourcemanager.frontdoor.models.Policies"
- "com.azure.resourcemanager.frontdoor.models.PreconfiguredEndpoint"
- "com.azure.resourcemanager.frontdoor.models.PreconfiguredEndpoints"
- "com.azure.resourcemanager.frontdoor.models.Profile"
- "com.azure.resourcemanager.frontdoor.models.Profile.Definition"
- "com.azure.resourcemanager.frontdoor.models.Profile.DefinitionStages"
- "com.azure.resourcemanager.frontdoor.models.Profile.DefinitionStages.Blank"
- "com.azure.resourcemanager.frontdoor.models.Profile.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.frontdoor.models.Profile.DefinitionStages.WithEnabledState"
- "com.azure.resourcemanager.frontdoor.models.Profile.DefinitionStages.WithEtag"
- "com.azure.resourcemanager.frontdoor.models.Profile.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.frontdoor.models.Profile.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.frontdoor.models.Profile.DefinitionStages.WithTags"
- "com.azure.resourcemanager.frontdoor.models.Profile.Update"
- "com.azure.resourcemanager.frontdoor.models.Profile.UpdateStages"
- "com.azure.resourcemanager.frontdoor.models.Profile.UpdateStages.WithEnabledState"
- "com.azure.resourcemanager.frontdoor.models.Profile.UpdateStages.WithTags"
- "com.azure.resourcemanager.frontdoor.models.Reports"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine.Definition"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine.DefinitionStages"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine.DefinitionStages.Blank"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine.DefinitionStages.WithRules"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine.Update"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine.UpdateStages"
- "com.azure.resourcemanager.frontdoor.models.RulesEngine.UpdateStages.WithRules"
- "com.azure.resourcemanager.frontdoor.models.RulesEngines"
- "com.azure.resourcemanager.frontdoor.models.Timeseries"
- "com.azure.resourcemanager.frontdoor.models.ValidateCustomDomainOutput"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.Definition"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.Blank"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.WithCustomRules"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.WithEtag"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.WithManagedRules"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.WithPolicySettings"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.WithSku"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.DefinitionStages.WithTags"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.Update"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.UpdateStages"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.UpdateStages.WithCustomRules"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.UpdateStages.WithEtag"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.UpdateStages.WithManagedRules"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.UpdateStages.WithPolicySettings"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.UpdateStages.WithSku"
- "com.azure.resourcemanager.frontdoor.models.WebApplicationFirewallPolicy.UpdateStages.WithTags"
metadata: {}
package: "com.azure.resourcemanager.frontdoor.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-frontdoor:1.0.0-beta.1
