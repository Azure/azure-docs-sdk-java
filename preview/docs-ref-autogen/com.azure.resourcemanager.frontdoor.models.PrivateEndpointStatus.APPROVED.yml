### YamlMime:JavaMember
uid: "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.APPROVED*"
fullName: "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.APPROVED"
  fullName: "com.azure.resourcemanager.frontdoor.models.PrivateEndpointStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointStatus."
  syntax: "public static final PrivateEndpointStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.frontdoor.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-frontdoor:1.0.0-beta.1
